torch/csrc/jit/codegen/cuda/scheduler/vectorize_helper.h

```diff
@@ -385,46 +385,77 @@ class TORCH_CUDA_CU_API ProjectedExtent {
 // Then tv0:
 //   maps on 7 with extent 7
 //   maps on 5 with extent 5
+//
+// This class is responsible for both computing the spanning tree and running
+// the spanning tree.
+//
+// In other words this class implements:
+//   MaxInfoSpanningTree::computeInfoC2P
+//     and
+//   MaxInfoSpanningTree::Propagator::propagateC2P
+//
+// The challenge here is the information we need for
+// MaxInfoSpanningTree::computeInfoC2P is the same information we need to
+// compute for MaxInfoSpanningTree::Propagator::propagateC2P
+//
+// We could compute both of these passes at the same time, only saving the
+// result produced from processing the edge that's chosen from
+// MaxInfoSpanningTree::computeInfoC2P while processing based on
+// MaxInfoSpanningTree::Propagator::propagateC2P. Hover, this would require
```

Comment:
nit: You meant this?
```suggestion
// MaxInfoSpanningTree::Propagator::propagateC2P. However, this would require
```